packages/esm-patient-chart-app/src/clinical-views/utils/index.ts

```diff
@@ -0,0 +1,99 @@
+import { getConceptFromMappings, getObsFromEncounter } from './helpers';
+import { type ColumnDefinition, type ConfigConcepts, type EncounterTileColumn, type MenuCardProps } from '../types';
+
+const calculateDateDifferenceInDate = (givenDate: string): string => {
+  const dateDifference = new Date().getTime() - new Date(givenDate).getTime();
+  const totalDays = Math.floor(dateDifference / (1000 * 3600 * 24));
+  return `${totalDays} days`;
```

Comment:
Seems like this could be:

```suggestion
   return `${dayjs().diff(dayjs(givenDate), 'days')} days`
```

Reply:
Also, the `days` here needs to be translatable. I'd probably have this function calculate the difference in days and then use i18next's format capabilities.